[2025-02-04T06:43:55.284+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table manual__2025-02-04T06:43:50.293731+00:00 [queued]>
[2025-02-04T06:43:55.301+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table manual__2025-02-04T06:43:50.293731+00:00 [queued]>
[2025-02-04T06:43:55.302+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2025-02-04T06:43:55.302+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2025-02-04T06:43:55.302+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2025-02-04T06:43:55.326+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): create_table> on 2025-02-04 06:43:50.293731+00:00
[2025-02-04T06:43:55.332+0000] {standard_task_runner.py:55} INFO - Started process 7075 to run task
[2025-02-04T06:43:55.339+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'create_table', 'manual__2025-02-04T06:43:50.293731+00:00', '--job-id', '17', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpew_lyfy8']
[2025-02-04T06:43:55.343+0000] {standard_task_runner.py:83} INFO - Job 17: Subtask create_table
[2025-02-04T06:43:55.452+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.create_table manual__2025-02-04T06:43:50.293731+00:00 [running]> on host fb1f760c5ca3
[2025-02-04T06:43:55.571+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2025-02-04T06:43:50.293731+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2025-02-04T06:43:50.293731+00:00
[2025-02-04T06:43:55.587+0000] {base.py:71} INFO - Using connection ID 'postgres' for task execution.
[2025-02-04T06:43:55.593+0000] {sql.py:315} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS users (
                firstname TEXT NOT NULL,
                lastname TEXT NOT NULL,
                country TEXT NOT NULL,
                username TEXT NOT NULL,
                password TEXT NOT NULL,
                email TEXT NOT NULL
                );
            , parameters: None
[2025-02-04T06:43:55.599+0000] {postgres.py:96} INFO - NOTICE:  relation "users" already exists, skipping

[2025-02-04T06:43:55.614+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=create_table, execution_date=20250204T064350, start_date=20250204T064355, end_date=20250204T064355
[2025-02-04T06:43:55.643+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2025-02-04T06:43:55.699+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
